tests/integration/test_workspace_io.py

```diff
@@ -73,4 +70,5 @@ def test_serialize_workspace(example, filename, monkeypatch):
     # TODO (midnighter): Use `from_orm` like `.construct` bypassing validation. (
     #  Requires a pull request on pydantic.)
     expected = WorkspaceIO.from_orm(Workspace.load(path))
-    assert WorkspaceIO.from_orm(example.main()) == expected
+    actual = WorkspaceIO.from_orm(example.main())
+    assert json.loads(actual.json()) == json.loads(expected.json())
```

Comment:
Why not use `.dict()` here? Or do you definitely want to load the JSON? In that case you could change it to:

```suggestion
    assert json.loads(actual.json()) == json.load(path)
```

Reply:
It's actually what I meant to do since our default values are not populated exactly like in the original examples. Mostly because the empty objects are populated in the C# SDK, but not default values, a behavior that I didn't find an easy way to reproduce exactly.

So, just to understand, the test that I'm doing is not perfect since it manipulates the expected JSON... But for now, that's what we have. I think that we should dive deeper into unit tests to make sure the functionality matches.

Reply:
My point is that:

```py
json.loads(actual.json())
```

is the same as

```py
actual.dict()
```